### PR TITLE
add the missing index.html in generated htmls

### PR DESCRIPTION
**Pull request by tom-shan**

### CODE REVIEW

File: libpip2pi/commands.py [hunk 1]
```diff
@@ -373,7 +373,7 @@ def _dir2pi(option, argv):
             shutil.copy2(pkg_filepath, symlink_target)
 
         if pkg_name not in processed_pkg:
-            pkg_index += "<a href='%s/'>%s</a><br />\n" %(
+            pkg_index += "<a href='%s/index.html'>%s</a><br />\n" %(
```
Comment (pombredanne) [inline libpip2pi/commands.py:376]: This may fail if you have a server-generated  index that's under some other name?

Comment (tom-shan) [reply libpip2pi/commands.py:376]: you are right, also this change will conflict with the "standard"(https://www.python.org/dev/peps/pep-0503/)

actually my initial intention was to support deploying pypi package repository by using "object storage service" which does't support "directory index".

**Comment by tom-shan** [comment]
I think using the capability of pip's "find-links" is a better way for my use case.
So I'm going to close this PR

**Comment by pombredanne** [comment]
FYI I found out that `pip --find-links` with `--no-index` (which I use extensively with scancode-toolkit) has some issues wrt. to caching and reusing of wheels by pip which `--index` does not and this can require more network operations slowing down some installations.
I am considering pip2pi exactly for this reason.

**Comment by tom-shan** [comment]
> FYI I found out that `pip --find-links` with `--no-index` (which I use extensively with scancode-toolkit) has some issues wrt. to caching and reusing of wheels by pip which `--index` does not and this can require more network operations slowing down some installations.
> I am considering pip2pi exactly for this reason.

Thanks for your info, actually we only use "find-links" as our fallback if no proper packages found in the official index. I think for you use case, maybe you can ask for help in pypa/pip repo.

**Pull request closed without merging**